### PR TITLE
feat(onboarding): change url for onboarding wizard

### PR DESCRIPTION
**Pull request by scefali**
This PR sets the referrer for onboarding task URLs when doing the redirect

### CODE REVIEW

File: static/app/components/onboardingWizard/task.tsx [hunk 1]
```diff
@@ -71,7 +71,9 @@ function Task({router, task, onSkip, onMarkComplete, forwardedRef, organization}
     }
 
     if (task.actionType === 'app') {
-      navigateTo(`${task.location}?onboardingTask`, router);
```
Comment (evanpurkhiser) [inline static/app/components/onboardingWizard/task.tsx:74]: Is it OK that the `onboardingTask` query key has gone away?

Comment (scefali) [reply static/app/components/onboardingWizard/task.tsx:74]: @evanpurkhiser tbh I don't even understand what it does, it's not like we said `referrer=onboarding_task`, this seemed like a bug more than anything lol

Comment (evanpurkhiser) [reply static/app/components/onboardingWizard/task.tsx:74]: I think `onboardingTask` may have at one point cased it to open the onboarding task list?

Comment (scefali) [reply static/app/components/onboardingWizard/task.tsx:74]: @evanpurkhiser Not sure why you'd do that for an item coming from the onboarding task list. Anyways, no code is referencing it anymore.

Comment (evanpurkhiser) [reply static/app/components/onboardingWizard/task.tsx:74]: sgtm

Comment (evanpurkhiser) [reply static/app/components/onboardingWizard/task.tsx:74]: oh no idea lol

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**